docs/development.md

```diff
@@ -0,0 +1,29 @@
+# Development Notes
+
+## Arrow Functions
+
+This projet heavily prefers the use of the `function` keyword over the use of
+[arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).
+Arrow functions may be used for one line callbacks.
+If the RHS of the arrow function is more than a single expression,
+use the `function` keyword instead.
+
+Both the RHS and LHS of the arrow must be wrapped in parentheses.
+For example:
+```javascript
+/* Good */
```

Comment:
For these "Good"/"Bad", you can either use them as "section headers" and put a newline after them,
or you can use them inline and put it with a colon at the beginning of the line comment.

Like:
```
// Bad

// Missing parentheses on both sides.
numbers.map(number => number * 2);

...
```

or 

```
// Bad: Missing parentheses on both sides.
numbers.map(number => number * 2);
```